packages/start-slicemachine/src/StartSliceMachineProcess.ts

```diff
@@ -124,16 +126,22 @@ export class StartSliceMachineProcess {
 				userID: profile.shortId,
 				intercomHash: profile.intercomHash,
 			});
-		}
 
-		if (profile) {
 			await Promise.allSettled([
 				// noop - We'll try again when needed.
 				this._sliceMachineManager.user.refreshAuthenticationToken(),
 				// noop - We'll try again before uploading a screenshot.
 				this._sliceMachineManager.screenshots.initS3ACL(),
 			]);
 		}
+
+		try {
+			this._trackStart();
```

Comment:
Is there a reason we delay tracking to the end of the process? Since it happens in the background and shouldn't affect start up performance significantly, I wonder if we should move this immediately after initializing telemetry and plugins

Reply:
You're right it's even better now!